oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -593,8 +596,9 @@ void findSlotObjects(const std::string& boardObjPath,
     }
     catch (const std::exception& e)
     {
-        error("no cec slots under motherboard {BOARD_OBJ_PATH}",
-              "BOARD_OBJ_PATH", boardObjPath.c_str());
+        error(
+            "no cec slots under motherboard {BOARD_OBJ_PATH} ERROR={ERR_EXCEP}",
```

Comment:
Please change "no cec slots under" to "No cec slots found under"

Reply:
updated as per comment.